browser/style.css

```diff
@@ -532,6 +532,24 @@ fieldset {
   margin: 0px 0px 5px 0px;
 }
 
+.settings.destroy {
+    color: color(#013545 tint(70%));
+    background-color: transparent;
+    border: 0;
+    letter-spacing: normal;
+    line-height: 31px;
+    margin-bottom: 0;
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 0;
+    padding-bottom: 10px;
+    padding-left: 10px;
+    padding-right: 10px;
+    padding-top: 0;
+    height: 31px;
+    font-size: 14px;
+}
+
```

Comment:
All this was basically copied from the table's destroy column/row button, under CSS selector `table button.destroy`